app/models/manageiq/providers/redhat/infra_manager/refresh/refresher.rb

```diff
@@ -26,7 +26,12 @@ def collect_inventory_for_targets(ems, targets)
           end
         end
 
-        data[:ems] = {:api_version => inventory.service.version_string}
+        case ems.highest_allowed_api_version
+        when 3
+          data[:ems_api_version] = {:api_version => inventory.service.version_string}
```

Comment:
based on https://github.com/ManageIQ/manageiq/pull/15252/files#diff-090b07f911cee85c2acff15e6bda1032R21, the key name must be `:ems`